docs/reference/commandline/docker.md

```diff
@@ -1,14 +1,7 @@
----
-title: "Use the Docker command line"
-description: "Docker's CLI command description and usage"
-keywords: "Docker, Docker documentation, CLI, command line, config.json, CLI configuration file"
-aliases:
-  - /reference/commandline/cli/
-  - /engine/reference/commandline/engine/
-  - /engine/reference/commandline/engine_activate/
-  - /engine/reference/commandline/engine_check/
-  - /engine/reference/commandline/engine_update/
----
```

Comment:
Did we take care of these elsewhere?

Reply:
yes: https://github.com/docker/docs/pull/19794/files#diff-8810b23e77efed3141d1ae3306ef22281909376a94e958ba06b92f9df1fc9f1e